tools/README.md

```diff
@@ -0,0 +1,110 @@
+# Istio Load Testing User Guide
+This guide provides step-by-step instructions for using the `setup_perf_cluster.sh` load testing script.
+The script deploys a GKE cluster, an Istio service mesh, a GCE VM and runs [Fortio](https://github.com/istio/fortio/)
+on the VM and within the mesh. Fortio is used to perform load testing.
```

Comment:
Fortio is used to perform load testing, graph the results and as the backend echo server as well.